zipline/finance/trading.py

```diff
@@ -78,7 +78,7 @@ class TradingEnvironment(object):
     def __init__(
         self,
         load=None,
-        bm_symbol='^GSPC',
+        bm_symbol='SPY',
```

Comment:
If i may make one final recommendation on all the benchmark data as you program this.  As a 20+ year hedge fund professional I can attest that the benchmark is good to have and good business practice to include it in your studies and presentations, so do NOT remove it like some have suggested. However to save valuable programming time and provide a top notch solution when it comes to missing data chunks (like we currently have with the time series from Google) filling in the missing periods with the backfill function would be totally acceptable, so long as you alert the user.  This can be a third choice: if a file is only partially available online.  So  maybe 
 the decision tree here regarding benchmarks could be: a. See if available online, b. see if the user can upload a csv, 3, if available online with NaN backfill and d.  All else fails grab an old benchmark file and backfill the missing data for the time range and alert the user.  Finally, if you can, don't call it the S&P benchmark for file names and inside the code, call it the equity benchmark as you may choose to change it later on.  You guys are doing a stellar job here.  Thank you!